content/may2015/medication-statement.md

```diff
@@ -0,0 +1,28 @@
+---
+title: MedicationStatement | FHIR MAY 2015 BALLOT API
+---
+
+# MedicationStatement
+
+* TOC
+{:toc}
+
+## Search
+
+Search for MedicationStatements that meet supplied query parameters:
+
+    GET /MedicationStatement?:parameters
+
+### Parameters
+
+ Name                                                     | Required? | Type                                                             | Description
+----------------------------------------------------------|-----------|------------------------------------------------------------------|-----------------------------------------------------------------------------------
+`patient`                                                 | Y         | [`reference`](http://hl7.org/fhir/2015May/search.html#reference) | The identifier of a patient to list statements for. Example: `12345`
+`status`                                                  | N         | [`token`](http://hl7.org/fhir/2015May/search.html#token)         | The status of the medication statement, may be a list separated by commas.  Example: `in-progress,completed`
+`effectivedate`                                           | N         | [`date`](http://hl7.org/fhir/2015May/search.html#date)           | The date-time which should fall within the period the patient was taking (or not taking) the medication. Must be prefixed by '>='  Example: `>=2015-01-01`
```

Comment:
In the may ballot, 'ge' is not supported.  '>=' is an example of a supported prefix in the may ballot version (https://www.hl7.org/fhir/2015May/search.html#2.1.1.3.6), but in dstu 2 final this was changed to avoid url escaping issues and other confusion (http://hl7.org/fhir/search.html#prefix).